mce-fbdev.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file mce-fbdev.h
+ * Frame buffer device handling code for the Mode Control Entity
+ * <p>
+ * Copyright 2015 Jolla Ltd.
+ * <p>
+ * @author Simo Piiroinen <simo.piiroinen@jollamobile.com>
+ *
+ * mce is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * mce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mce.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MCE_FBDEV_H_
+# define MCE_FBDEV_H_
+
+# include <stdbool.h>
+
+# ifdef __cplusplus
+extern "C" {
+# elif 0
+} /* fool JED indentation ... */
```

Comment:
Sadly no. And I agree on the hmm.. part. But the other lesser editors do not have even decent z80 modes with clock cycle count helpers... Meaning the only way I can do semi-sane "autoindent whole file" is to have those  hacks in headers or spend time fixing the autoindent code - and I already have configured jed to handle the former automatically.